### DISCUSSION TITLE
Reconnect script

### DISCUSSION THREAD

**Discussion by webepg** [haugene/docker-transmission-openvpn#2373] [General]
Hi,

I installed JDownloader within your docker. It is running great except the thing with the reconnect when the download limit of a hoster is reached. I then would like to run a reconnect script to connect to an other VPN. How would I do that? Currently I'm restarting the container but this is not the best thing to do

**Comment by pkishino** [comment]
?why would you install jdownloader inide the container? that kinda goes against the whole point in using containers.
this container is for using transmission with openvpn, anything else you are better of looking for a different container

**Comment by webepg** [comment]
I replaced transmission with JDownloader. They have the same purpose. So there still is the same thing. I think even making the container unhealthy would help to restart it automatically
